python/pip_install/entrypoint.bzl

```diff
@@ -0,0 +1,60 @@
+# Copyright 2023 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"entry_point macro implementation for bzlmod. PRIVATE ONLY."
+
+load("//python/private:version_label.bzl", "version_label")
+
+def entry_point(*, pkg, packages, default_version, tmpl, script = None):
```

Comment:
Not sure if this is the preferred pattern, but it let's you unit test.

Reply:
I will always prefer code that you can unit test :)